### ISSUE TITLE
Not working on mt5

### ISSUE THREAD

**Issue by FX-Emperor** [sholafalana/MT5-MT4-Telegram-API-Bot#1]
I tried since it says MT5-MT4-Telegram-API-Bot by changing the name of ea from TelagramAlert.mt4 to TelegramAlert.mt5. It doesn't show up on mt5 and metaeditor shows 90 errors and 13 warnings. 

How can I make this work on mt5?

**Comment by FX-Emperor** [comment]
@sholafalana  It works sometimes on mt4 too. Most times it gave an error bad request, chat not found. So I had to take it down but the other one I'm testing sends too many order modify notifs so I like this one better. Just don't know how to fix that issue. I've removed the bot, checked its privacy, added again and tried but still the same. The other ea I'm testing is a demo version from the mql market.
